### ISSUE TITLE
Support data modifications

### ISSUE THREAD

**Issue by pheyvaer** [KNowledgeOnWebScale/walder#6]

**Comment by bjdmeest** [comment]
I assume this issue means 'if walder is running and the data changes, the data isn't refetched and the query doesn't re-run, even with --no-cache option'? If so: 👍  :)

**Comment by pheyvaer** [comment]
No, this means that Walder offers the option to alter the data in the knowledge graphs that it is using.

**Comment by sideshowtom** [comment]
Something in the readme about this, like what the problems are or if this is contemplated or if there are alternatives, would be great.
